packages/ckeditor5-emoji/docs/_snippets/features/emoji.html

```diff
@@ -1,7 +1,37 @@
-<script src="https://cdn.ckbox.io/ckbox/latest/ckbox.js"></script>
 <div class="ck ck-content" id="snippet-emoji">
-	<h3>Various emojis include:</h3>
-	<p>...</p>
+	<h3>Overview of Emojis</h3>
+
+	<p>
+		Emojis are small digital images or icons used to express emotions, ideas, or concepts in electronic communication. They
+		originated in Japan in the late 1990s, with the first set created by Shigetaka Kurita. Since then, emojis have evolved
+		into a global communication tool, becoming an essential part of online and mobile messaging. Emojis convey non-verbal
+		cues, adding emotional context or visual interest to text-based conversations. They help to bridge the gap left by the
+		absence of facial expressions and tone in digital communication.
+	</p>
+
+	<h3>Examples of Common Emojis</h3>
+
+	<ul>
+		<li>😊 Smiling Face: Used to convey happiness, friendliness, or gratitude. This emoji is often used in conversations to
```

Comment:
Perhaps we could add bold or italics effect to emoji names to visually separate them from description?

Reply:
Done in https://github.com/ckeditor/ckeditor5/pull/17713/commits/33bd7e61eb2e98c065e8fae387921e063bc78b4e.

![obraz](https://github.com/user-attachments/assets/4826f2b6-f3e3-4b62-8daa-5cad1a727377)